src/components/molecules/WorldCard/WorldCard.tsx

```diff
@@ -48,7 +48,11 @@ export const WorldCard: React.FC<WorldCardProps> = ({ world }) => {
         variant="dark"
         isLink
         disabled={!world.slug}
-        linkTo={adminWorldSpacesUrl(world.slug)}
+        linkTo={generateUrl({
+          route: ADMIN_IA_WORLD_PARAM_URL,
+          required: ["worldSlug"],
+          params: { worldSlug: world.slug },
+        })}
```

Comment:
I'm not a fan of using `generateUrl` everywhere. It's more verbose. It requires someone using it to write out which parts of the URL are required. It loses type validation. The thin wrapper function `adminWorldSpaceUrl` gives all of that.

Reply:
@colinhowe  It has already been done elsewhere https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1638790538446100

Reply:
Here is a quote of it:
>  generateUrl should be preferred to any other URL generator for your links, since we can upgrade it in the future, but for now provides a bit more explicit DX. Future upgrades might be adding a generic <T> for the params it expects and making that optional required array not necessary (if we parse the route itself for : and ? - simple regex does it)